pkg/controller/workload/workload_processor.go

```diff
@@ -520,6 +520,11 @@ func (p *Processor) handleWorkload(workload *workloadapi.Workload) error {
 		newWorkloadAddresses := workload.GetAddresses()
 		oldWorkloadAddresses := oldWorkload.GetAddresses()
 
+		if len(newWorkloadAddresses) == 0 {
+			log.Warnf("new workload address: %s/%s is nil", workload.Namespace, workload.Name)
```

Comment:
I feel should filter workload without address in the entrance since we donot support dns service in dualengine now?